### PR TITLE
utils change

### PR DESCRIPTION
**Pull request by sgnanase**

### CODE REVIEW

File: vendorsetup.sh [hunk 1]
```diff
@@ -30,14 +30,15 @@ function lunch
...
-        mixinup -s $product_dir/mixins.spec
+        #mixinup -s $product_dir/mixins.spec
+        mixinup
     fi
 }
 
 # Get the exact value of a build variable.
 function get_build_var()
 {
-    if [ "$1" = "COMMON_LUNCH_CHOICES" ]
```
Comment (shyjumon-n) [inline vendorsetup.sh:40]: I think its a typo ?

**Review by shyjumon-n — CHANGES_REQUESTED** [review]
pls check typo

File: vendorsetup.sh (hunk 1 above)
Comment (sgnanase) [reply vendorsetup.sh:40]: Its a temporary change, ill remove this function from script soon

**Review by shyjumon-n — APPROVED** [review]
LGTM

**Review by balajim001 — APPROVED** [review]
LGTM

**Pull request merged**